### PR TITLE
gossip: add utilities for config generation

### PR DESCRIPTION
**Pull request by daviddrysdale**
- ./gossip/configpb/from_ct generates a template Hub configuration
   based on a Certificate Transparency log list.
 - ./gossip/testdata/to_proto emits protobuf stanzas describing the
   private keys held in PEM files.

### CODE REVIEW

File: gossip/testdata/to_proto/main.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			pwdDer, err := x509.DecryptPEMBlock(block, []byte(*password))
+			if err != nil {
+				glog.Errorf("%v: failed to decrypt: %v", arg, err)
+				continue
+			}
+			keyDER = pwdDer
+		}
+
+		privProto := keyspb.PrivateKey{Der: keyDER}
+		fmt.Printf("\tprivate_key: {\n")
```
Comment (gdbelvin) [inline gossip/testdata/to_proto/main.go]: Might be easier to use proto any here?

Comment (daviddrysdale) [reply gossip/testdata/to_proto/main.go]: Good idea, done.

File: gossip/testdata/to_proto/main.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+			glog.Errorf("%v: failed to parse private key: %v", arg, err)
+			continue
+		}
+
+		privProto, err := ptypes.MarshalAny(&keyspb.PrivateKey{Der: keyDER})
+		if err != nil {
+			glog.Errorf("%v: failed to marshal private key as Any: %v", arg, err)
+		}
+
+		fmt.Printf("\tprivate_key: {\n")
```
Comment (gdbelvin) [inline gossip/testdata/to_proto/main.go]: Should this also be put in it's proto and marshal the whole thing?

Comment (daviddrysdale) [reply gossip/testdata/to_proto/main.go]: Good idea, I forgot that the other unused fields would be silently omitted from the output.

**Pull request merged**